.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  pull_request:
+  push:
+  release:
+    types: published
+
+jobs:
+  build:
+    runs-on: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install dependencies
+      run: |
+        sudo apt install -y python3-pytest
```

Comment:
I have intermittent failures with `apt` on actions if not doing `apt update` first.
However, in this case, I would consider doing `pip install pytest` instaed

Reply:
Why?

Reply:
Re: using `pip` instead of `apt` -- one gets the latest Pytest release, but the real motivation here is to have the Actions image startup and run the test faster, by avoiding slower `apt update`. Just like for a Linux laptop, once in a while `apt update` is needed because the distro changed something such that `apt install` won't work until `apt  update`.  This type of problem can persist at least for hours until Actions staff update their Ubuntu image cache.

Reply:
Unfortunately I need apt anyway for clang-tidy and other packages.